janus_core/calculations/md.py

```diff
@@ -1044,6 +1074,89 @@ def _set_target_temperature(self, temperature: float):
         else:
             raise ValueError("Temperature set for ensemble with no thermostat.")
 
+    def _init_progress_bar(self) -> ProgressBar:
+        """
+        Initialise MD progress bar.
+
+        Returns
+        -------
+        progress_bar
```

Comment:
```suggestion
        ProgressBar
```
See: https://numpydoc.readthedocs.io/en/latest/format.html#returns